engine/src/main/java/org/hibernate/validator/internal/constraintvalidators/bv/MaxValidatorForDuration.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Hibernate Validator, declare and validate application constraints
+ *
+ * License: Apache License, Version 2.0
+ * See the license.txt file in the root directory or <http://www.apache.org/licenses/LICENSE-2.0>.
+ */
+package org.hibernate.validator.internal.constraintvalidators.bv;
+
+import java.time.Duration;
+import java.time.temporal.ChronoUnit;
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+import javax.validation.constraints.Max;
+
+/**
+ * Check that the validated {@link Duration} is less than or equal to the minimum
+ * value specified.
+ *
+ * @author Marko Bekhta
+ */
+public class MaxValidatorForDuration implements ConstraintValidator<Max, Duration> {
+
+	private Duration duration;
```

Comment:
Maybe call them `maxDuration` (and `minDuration` for the `MinValidatorForDuration`). It would be easier to apprehend the comparison below.